actionmailbox/lib/tasks/install.rake

```diff
@@ -1,20 +1,6 @@
 # frozen_string_literal: true
 
-namespace :action_mailbox do
-  # Prevent migration installation task from showing up twice.
-  Rake::Task["install:migrations"].clear_comments
-
-  desc "Copy over the migration"
-  task install: %w[ environment run_installer copy_migrations ]
-
-  task :run_installer do
-    installer_template = File.expand_path("../rails/generators/installer.rb", __dir__)
-    system "#{RbConfig.ruby} ./bin/rails app:template LOCATION=#{installer_template}"
-  end
-
-  task :copy_migrations do
-    Rake::Task["active_storage:install:migrations"].invoke
-    Rake::Task["railties:install:migrations"].reenable # Otherwise you can't run 2 migration copy tasks in one invocation
-    Rake::Task["action_mailbox:install:migrations"].invoke
-  end
+desc "Copy over the migration"
```

Comment:
I wonder if the description should be more generic since it also creates `application_mailbox.rb`. Just a thought, I have no specific text in mind.

Reply:
How about changing it to `Installs Action Mailbox and ensures Active Storage is set up`?

Reply:
May I suggest: `Installs ActionMailbox and its dependencies`?

Reply:
Thanks, @vinistock . Changed the description as per your suggestion. I'll squash my commits in the end.

Reply:
Looks good to me!